src/test/java/com/hubspot/imap/utils/parsers/AtomOrStringParserTest.java

```diff
@@ -21,6 +23,9 @@
   private static final byte[] QUOTED_RESULT = Arrays.copyOfRange(QUOTED, 1, QUOTED.length-1);
   private static final byte[] ESCAPED_QUOTES = "\"This contains \\\"quotes\\\"\"".getBytes(StandardCharsets.UTF_8);
   private static final byte[] ESCAPED_QUOTES_RESULT = Arrays.copyOfRange(ESCAPED_QUOTES, 1, ESCAPED_QUOTES.length - 1);
+  private static final byte[] LITERAL = "{10}\nabcdefghij".getBytes(StandardCharsets.UTF_8);
```

Comment:
To my previous comment, can we try this with a real buffer? Not sure what it will actually look like but that way we will know if something follows the literal string usually or not